Knightj-valheim-build-treehouse

jknight6 opened the pull request:
This PR is for the How to Build a Treehouse procedure.

Ademola-Adepoju reviewed (approved):
This looks good! I like how straightforward the procedure is. From a psychological angle, it makes the reader feels more relaxed that what they're trying to do isn't hard, whether it is hard in reality or not.

sezito2 reviewed (approved):
I think it would be easier to read if instead of subheadings, your steps were in an ordered list instead. I like the inclusion of images

jknight6 committed:
Add my project and begin work on procedure one

This commit adds the project folders for my procedures, and adds the first sections of my first procedure, How to Defeat Bonemass in Valheim.

jknight6 committed:
Finish writing procedure one

This commit finishes the first draft for procedure one and prepares it for review.

jknight6 committed:
Images, About and Prerequisites

This commit covers the images for this procedure, as well as the About and Prerequisites section.

jknight6 committed:
How to of the procedure

This commit wraps up the procedure with the how-to steps and conclusion.

jknight6 committed:
First commit, adding top picture and About section

This first commit adds the title, picture, and About sections.

jknight6 committed:
Add Prerequisites section

This commit adds the prerequisites section.

jknight6 committed:
Add procedure and conclusion sections

This commit adds the procedure and conclusion elements for this procedure.

Alli-Wray commented on knightj/p-valheim-build-treehouse.md:
```diff
@@ -0,0 +1,37 @@
+
+As mentioned above, building a treehouse is a challenging project. In order to be successful and build a treehouse you're proud of, you should have access to the stonecutter. It would also enhance your experience if you had access to iron materials.
+
+If you don't have much experience building structures in Valheim, this would be a difficult first build. Try building different types of buildings in the meadows and black forest before giving this a shot.
```
This is a great note to make, and I like the location of it within the text. I would consider this to potentially be an example of alerting, so adding a visual cue, such as an asterisk or something similar, might help it stand out.

jknight6 replied:
I blocked it off and made it bold to help it stand out from the text above and also from the other types of alerts in this procedure.

Alli-Wray commented on knightj/p-valheim-build-treehouse.md:
```diff
@@ -0,0 +1,37 @@
+
+>*Note that you can tell which trees are indestrucible because they will have green guck on their trunks, and hitting them with any weapon will alert you that it is too hard to be cut down.
+
+2. Build a platform at ground level that will serve as a foundation from which to build the structure.
```
I like the use of imperative verbs here. My only remaining suggestion for this procedure would be to add some images for certain steps. For example, I think it could be helpful to show an image of an example platform for step 2 here.

jknight6 replied:
Great idea, added an image

Alli-Wray commented on knightj/p-valheim-build-treehouse.md:
```diff
@@ -0,0 +1,37 @@
+
+## About Building a Swamp Treehouse
+
+Building a treehouse is a challenging build project in Valheim. The mechanics around building, particularly the support mechanics, make tall structures complicated to balance. Even with these challenges, trees in the swamp are uniquely positioned for building fun, interesting houses.
```
You mention this a few times, so I'm wondering if there are some specific examples of obstacles the user may come across when building. Is it possible for the structure to collapse if built incorrectly? Are there certain materials that shouldn't be used together? If so, I think adding some warnings would be super helpful!

jknight6 replied:
There are - structures can collapse if they aren't connected directly enough with the ground, that's the biggest challenge. I tried to make this procedure aimed at intermediate to advanced players, so they should be fairly familiar with the building mechanics and challenges. If the rest of the procedure seems more geared towards beginners, maybe I need to address that to make it more consistent.

Alli-Wray commented on knightj/p-valheim-build-treehouse.md:
```diff
@@ -0,0 +1,37 @@
+2. Build a platform at ground level that will serve as a foundation from which to build the structure.
+3. Build ladders or stairs from the platform up to where you want to start building your treehouse.
+    - You can snap the side of the stairs to the tree to give it ground-level stability, allowing you to build as high up the tree as you want.
+4. Place corewood or iron beams as the framework for the floor of your treehouse, branching out from the trunk.
```
Is there a particular method to this, or is it up to the user? This step seems somewhat vague to me, but I'm not familiar with Valheim so I could be out of the loop here.

jknight6 replied:
There isn't a particular method, it kinda has to be vague because everyone's building style is different and there are lots of viable options so I'm just trying to give the general framework so users can take it from there.

Alli-Wray reviewed (commented):
I've left a few comments about what I think could be added to enhance this procedure. Overall, I think the layout and formatting looks great, and you have good staging and coaching so far.

stevesynk commented on knightj/p-valheim-build-treehouse.md:
```diff
@@ -6,4 +6,10 @@
+
+## Prerequisites
+
+As mentioned above, building a treehouse is a challenging project. In order to be successful and build a treehouse you're proud of, you should have access to the stonecutter. It would also enhance your experience if you had access to iron materials.
```
This feels a bit repetitive, since you just mentioned that it is challenging about 2 sentences back. Here's a more concise rewording to consider for combining the first two sentences in this section:

"In order to be successful in the challenging build, you will need access to the stonecutter."

jknight6 replied:
Agreed - I'll make the edit.

jknight6 replied:
I decided to just cut the first sentence, since I mentioned the challenging nature in the introduction it is redundant and I don't think adds anything to the prerequisites section to mention it again.

stevesynk commented:
Nicely done overall. Headers and staging were good. Procedures were smooth and clearly worded, although I left some nitpicky comments about phrasing.

The only thing is right now I am not seeing the images in this procedure, and when I follow the links I get a "not found."

jknight6 committed:
Respond to comments and make edits from review

This commit contains comment responses and edits in response to peer review.

jknight6 committed:
Add reflection and user scenarios, and update readme

This commit adds the reflection and user scenarios to the project, and updates the readme.

lingeringcode committed:
Feedback from lindgren
